pkg/cmd/clustertask/describe.go

```diff
@@ -34,6 +34,7 @@ import (
 )
 
 const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .ClusterTask.Name }}
+{{decorate "bold" "Description"}}:	{{ .ClusterTask.Spec.Description }}
```

Comment:
Should this be hidden if not specified? If the `Description` is empty, I think it can be excluded.

Reply:
yup, makes sense, I will update

Reply:
done